### PR TITLE
Live Code - support more components 1

### PR DESCRIPTION
**Pull request by M-i-k-e-l**
## Description
Live Code - support more components 1
Go over each commit alone (or don't)

ActionBar - floating over the result (added a margin to the example to solve it)
Checkbox - the checked v is not showing

## Changelog
None

## Additional info
None

### CODE REVIEW

File: src/components/checkbox/checkbox.api.json [hunk 1]
```diff
@@ -42,6 +42,13 @@
     }
   ],
   "snippet": [
-    "<Checkbox value={false$1} onValueChange={() => console.log('value changed')$2}/>"
+    "function Example(props) {",
+    "  const [value, setValue] = useState(false$1);",
+    "  return (",
+    "    <div>",
+    "      <Checkbox value={value$2} onValueChange={setValue$3}/>",
```
Comment (adids1221) [inline src/components/checkbox/checkbox.api.json:49]: When selecting the check box there is an issue with the V icon, in the webDemo it works.

File: src/components/slider/slider.api.json [hunk 2]
```diff
@@ -85,13 +85,5 @@
     },
     {"name": "testID", "type": "string", "description": "The component test id"},
     {"name": "migrate", "type": "boolean", "description": "Temporary prop required for migration to the Slider's new implementation"}
-  ],
-  "snippet": [
-    "<Slider",
```
Comment (adids1221) [inline src/components/slider/slider.api.json:90]: Why removing the Slider snippet ?

**Review by adids1221 — COMMENTED** [review]
Left a small comment about the Slider snippet.
Also I notice a general issue when trying to change some value is throw an error (let's sync about it).

File: src/components/slider/slider.api.json (hunk 2 above)
Comment (M-i-k-e-l) [reply src/components/slider/slider.api.json:90]: We could leave it but IMO we should not support components we want to deprecate (we have the `Incubator.Slider` that will be added in another PR).

**Comment by M-i-k-e-l** [comment]
> Left a small comment about the Slider snippet. Also I notice a general issue when trying to change some value is throw an error (let's sync about it).

I assume the errors we see when we edit (big annoying overlay) will not be there in production (or we'll have to fix it :)

**Review by adids1221 — APPROVED** [review]

**Pull request merged**